R/filters.R

```diff
@@ -234,17 +234,70 @@ get_calibrate_barchart_defaults <- function(filters) {
 }
 
 get_comparison_barchart_defaults <- function(output, filters) {
+  area_default <- get_area_level_filter_option(output)
   list(
-    indicator_id = scalar("prevalence"),
-    x_axis_id = scalar("age"),
-    disaggregate_by_id = scalar("source"),
-    selected_filter_options = list(
-      area = get_area_level_filter_option(output),
-      quarter = get_selected_mappings(filters, "quarter")[2],
-      sex = get_selected_mappings(filters, "sex")[1],
-      age = get_selected_mappings(filters, "age",
-                                  naomi::get_five_year_age_groups()),
-      source = get_selected_mappings(filters, "source")
+    list(
```

Comment:
Don't we also need to retain the indicator to select by default?

Reply:
I've proposed we just use the first see https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3843/Improve-comparison-plot-usability#focus=Comments-97-18340.0-0, but open to making that more explicit

Reply:
Ok so sounds like this will mean we cannot specify the default indicator separately, it will always be the first. We could do something like

```
"barchart": {
  "defaultSelections": [
     {..}
  ],
  "defaultIndicator": "prevalence"
}
```

Could even make default selections an object and have
```
{
  "prevalence": {...},
  "art_current": {...},
  ...
}
```